docs/concepts/deposits-withdrawals.md

```diff
@@ -0,0 +1,116 @@
+
+Once an asset is deposited, it's available in your general account, from which it then may be held in other account types depending on what you use the assets for. 
+
+:::tip Tools you'll need
+To deposit or withdraw, you'll need a [Vega Wallet](../tools/vega-wallet/index.md) and an Ethereum wallet, such as MetaMask. 
+:::
+
+:::note Read more 
+[Accounts](./accounts.md): Learn about different accounts that your assets can be held in. 
+:::
+
+## Deposits
+Only assets that have been proposed and enacted through governance can be used on the Vega network.
+
+To use assets on the Vega network, for example to take part in markets, you'll need to deposit it: 
+* The assets need to be available in an Ethereum wallet that you have access to
+* You'll need to approve the deposit to the Ethereum network, which gives the Vega bridge access to that asset in your Ethereum wallet
+* After approving, choose an amount to deposit. This deposit will have to accepted on the Ethereum network and the Vega network before the assets can be used
+
+
+:::info Try it out
+Deposit assets using the [Vega Console trading interface](https://console.fairground.wtf), in the Portfolio section.
+:::
+
+The first assets that will be available for interacting with markets on Vega will be ERC-20 assets, the only bridge currently available. An asset's ERC-20 token contract needs to be available on the bridge before it can be used, which happens in the governance and enactment process for new assets.
+
+An asset can then be deposited into the ERC20 bridge contract. The funds in that smart contract will then be made available to the user's chosen Vega public key. 
+
+Note: Associated and deposited are not equivalent, as deposited tokens are held within the ERC-20 bridge contract, and associated tokens stay in an Ethereum wallet or in the vesting contract.
+
+### Lifetime deposit limits 
+During alpha mainnet, the ERC-20 bridge smart contract limits how much can ever be deposited from an Ethereum address. This is done in an abundance of caution, to assure users in the face of recent bridge hacks that they would have only a small amount at risk at any point. 
+
+If, however, a user wanted to bypass those limits and understood the risks to their assets, they could run `exempt_depositor()` on the [ERC-20 bridge contract](./../api/bridge/contracts/ERC20.md) for each asset, after which transactions greater than deposit limit for the asset would be allowed.
+
+### Depositing ERC-20 assets
+Deposits go through the ERC0-20 bridge smart contract. Every type of asset supported by and voted into Vega will have a bridge, but for the time being there is only an ERC-20 bridge.
+
+When a participant wants to deposit assets onto a Vega key, they need to call a deposit function on the ERC20 bridge that contains how much of the specified asset the Vega key will receive.
+
+Once deposited, the assets are held in an asset pool for security and to make contract updates easier/less risky.
```

Comment:
```suggestion
Once deposited, the assets are held in the asset pool smart contract (WE SHOULD LINK TO THIS) for security (the bridge can be disconnected by a quroum of validators to prevent it accessing funds if a bug is found and withdrawals need to be stopped temporarily) and to make contract updates easier and less risky.
```